### PR TITLE
Section description (Assessment tool)

### PR DESCRIPTION
**Pull request by ElminaIusifova**
Added description for each capability. 
#1406

### CODE REVIEW

File: _pages/en/capabilities.md [hunk 1]
```diff
@@ -152,6 +152,9 @@ To achieve goals of efficient cooperation, collaboration, and bug-free code each
 
 ### Section 1. VERSION CONTROL AND CODE MAINTAINABILITY
 
+*Description: Learn how to implement the right version control practices for reproducibility and traceability.
```
Comment (gcharest) [inline _pages/en/capabilities.md:155]: Should we say "the right" or go for something a bit smoother like "better"?

File: _pages/en/capabilities.md [hunk 2]
```diff
@@ -245,6 +251,9 @@ This excerpt from to the [DORA DevOps capabilities guides](https://cloud.google.
 
 ### Section 3. CONTINUOUS TESTING
 
+*Description: Improve software quality by building reliable automates test suits and performing all kinds of testing throughout the software delivery lifecycle.
```
Comment (gcharest) [inline _pages/en/capabilities.md:254]: I think we meant "automated" instead of "automates"? :thinking:

File: _pages/en/capabilities.md [hunk 3]
```diff
@@ -326,11 +335,15 @@ Unit tests should not depend on external data. As defined by the test automation
 
 ### Section 4. ARCHITECTURE
 
+*Description: Learn about moving from a tightly coupled architecture to service-oriented and micro service architectures without re-architecting everything at once.
```
Comment (gcharest) [inline _pages/en/capabilities.md:338]: Nothing to change here, just super thrilled that this section exists :smile:

**Review by gcharest — COMMENTED** [review]
Good work! Just a few little things to confirm and this would be good to merge imho :smile:

File: _pages/en/capabilities.md (hunk 2 above)
Comment (ElminaIusifova) [reply _pages/en/capabilities.md]: fixed all the above comments, thanks

**Review by smellems — APPROVED** [review]

**Pull request merged**